common/src/main/java/org/dash/wallet/common/ui/FancyAlertDialog.kt

```diff
@@ -30,13 +30,14 @@ import androidx.annotation.DrawableRes
 import androidx.annotation.StringRes
 import androidx.fragment.app.DialogFragment
 import androidx.fragment.app.FragmentManager
-import kotlinx.android.synthetic.main.fancy_alert_dialog.*
```

Comment:
`kotlin-parcelize` plugin conflicts with `kotlin-android-extensions`, so to get rid of the latter this dialog is moved to view bindings.